src/styles/container.scss

```
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
+}
```

I would prefer a grid
```suggestion
.container {
    display: grid;
    grid-template-columns: repeat(auto-fill, 200px);
    gap: $gapUnderCards $gapBetweenCards;
    margin: 50px auto 0;
    max-width: mainContainerSize($cardWidth, $gapBetweenCards);
}
```